_pages/en/it-picture-medium-term.md

```diff
@@ -132,11 +146,20 @@ En conclusion, ...
 - https://cloud.google.com/solutions/devops/devops-measurement-wip-limits
 - https://cloud.google.com/solutions/devops/devops-measurement-visual-management
 
+Business lines understand their accountability in sustaining the IT systems used to enable their business.
+This means part of their budget is set to assure technical debt is measured, exposed to the business line, and handled in a timely manner
+
+Operations have moved towards a Site Reliability Engineering (SRE) practice where operations is treated like a software practice.
```

Comment:
I think we shouldn't call it SRE since it looks it's more that monitoring, it's like Devops.  We should remove all mentions of DevOps (execpt for the reference to DORA).  Agile would be acceptable, maybe, but not too much.  Cloud is only mentioned in GC-wide section

For this section we should also think about how we could be measuring lead time to get devices and how well those devices are performing. Access to the network, ..

Reply:
Reworded.
Though I'd like to keep the business lines statement about budget. It's part of the Technical Debt team that is looking for sustainable lifecycle management of applications. Also what EA is exploring in the ~5% "tax" on projects should a certain application starts becoming out dated.